### PR TITLE
Make order columns searchable

### PR DESCRIPTION
**Pull request by islean**
## Description

This PR adds support for searching for both order_id and ticket_id in the Order table.

### Added

-

### Changed

- id and ticket_id are searchable fields in Order.

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
Works in stage: 
image

File: cg/server/admin.py [hunk 1]
```diff
@@ -454,6 +454,7 @@ class OrderView(BaseView):
 
     column_default_sort = ("order_date", True)
     column_editable_list = ["is_delivered"]
+    column_searchable_list = ["id", "ticket_id"]
```
Comment (seallard) [inline cg/server/admin.py:457]: Is the search already done against the ticket id in the backend?

**Review by seallard — APPROVED** [review]
Looks good 👍 Only uncertain whether we have implemented support for this in the backend

File: cg/server/admin.py (hunk 1 above)
Comment (islean) [reply cg/server/admin.py:457]: This is the backend 💪

**Pull request merged**